ga4gh/datamodel/variants.py

```diff
@@ -719,6 +716,17 @@ def getVariantAnnotationId(self, gaVariant, gaAnnotation):
             gaVariant.start, md5)
         return str(compoundId)
 
+    def getVariantId(self, gaVariant):
```

Comment:
No, as it uses the variant set's compound ID as opposed to the annotation set's.